src/components/UserActions/UserActions.scss

```diff
@@ -84,10 +84,26 @@ $search-ui-dropdown-header-color: #1d4f76;
 
         li {
```

Comment:
@louis-bompart I think that's a good example for your open question.
The component needs specific styling for `li` here. 
So you create a rule with `.class > li` and not just `li` for example which Salesforce may control. That way you risk less conflicts by controlling what you need.

Reply:
I agree 👍 
However I'm wondering, if that `text-indent: -1em` etc is because of Salesforce, or not.

Reply:
What's triggering is more the `text-indent: -1em + padding-left: 1em`.

Reply:
The indent rules aren't Salesforce specific. I put them there so the text on multi-line items wouldn't start after the icon "bullet" I think it looks nicer this wa but that's open to discussion as well.